--- rtorrent-0.4.3/configure	2006-01-17 12:25:20.000000000 -0500
+++ newrtorrent/configure	2006-03-01 06:58:05.000000000 -0500
@@ -4050,7 +4050,7 @@
       hex_ver=`curl-config --vernum | tr 'a-f' 'A-F'`
       ok=`echo "ibase=16; if($hex_ver>=$check_hex) $hex_ver else 0" | bc`
 
-      if test x$ok != x0; then
+      if test x"$ok" != x0; then
         my_cv_curl_vers="$ver"
         echo "$as_me:$LINENO: result: $my_cv_curl_vers" >&5
 echo "${ECHO_T}$my_cv_curl_vers" >&6
